cura/PrinterOutput/NetworkedPrinterOutputDevice.py

```diff
@@ -178,7 +178,7 @@ def put(self, target: str, data: str, onFinished: Optional[Callable[[Any, QNetwo
         if onFinished is not None:
             self._onFinishedCallbacks[reply.url().toString() + str(reply.operation())] = onFinished
 
-    def get(self, target: str, onFinished: Optional[Callable[[QNetworkReply], None]]) -> None:
+    def get(self, target: str, onFinished: Optional[Callable[[Any, QNetworkReply], None]]) -> None:
```

Comment:
These changes are incorrect. The signature of a *bound* method in Python doesn't have the `self` parameter.
```
class Foo:
    def bar(self, some_int: int) -> None:
        # blah blah blah

my_foo = Foo()
bar_method = my_foo.bar
```

`bar_method` is now effectively a function which takes one parameter, an int. The `self` part of the original method is "pre-filled".

Reply:
Except that all type hinting completely trip up if you don't set it like that, which defeats the purpose of the hinting. So we can write it correctly and train users to ignore the type hinting or write it incorrect and train people to trigger at the warnings.

Reply:
Which application is being tripped up? mypy should be fine with it.

Reply:
pycharm, which runs mypy.

Reply:
Pycharm has its own checker, it doesn't run mypy. Mypy expects the first parameter to be omitted from the signature.